src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -89,11 +89,19 @@ public WorkspaceView()
         void OnWorkspaceViewLoaded(object sender, RoutedEventArgs e)
         {
             DynamoSelection.Instance.Selection.CollectionChanged += new NotifyCollectionChangedEventHandler(OnSelectionCollectionChanged);
+            ViewModel.RequestShowInCanvasSearch += ShowInCanvasControl;
+
+            var searchViewModel = new SearchViewModel(this.ViewModel.DynamoViewModel, this.ViewModel.DynamoViewModel.Model.SearchModel);
```

Comment:
For this I have made [a minor maintenance task](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7450), very minor but just want to see that happen :)